kroxylicious-docs/docs/_modules/configuring/con-configuring-vc-target.adoc

```diff
@@ -0,0 +1,62 @@
+////
+  // Copyright Kroxylicious Authors.
+  //
+  // Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+////
+:_mod-docs-content-type: CONCEPT
+
+[id='con-configuring-vc-target-{context}']
+= Configuring target cluster
+
+[role="_abstract"]
+The target cluster specifies the physical upstream Kafka cluster which is proxied by the `my-cluster-proxy` virtual cluster.
+
+The target cluster configuration has 3 properties.
+
+1. `bootstrapServers`: The bootstrap addresses of the physical Kafka cluster being proxied. You can specify multiple addresses by providing a comma separated list. If you want to bootstrap to individual Kafka brokers, then having multiple addresses helps spread the bootstrapping load and also provides better reliability.
++
+NOTE: If the target cluster is a *Strimzi* managed Kafka cluster, you can specify the address of the *bootstrap service* which takes care of distributing the requests.
```

Comment:
I think this needs to be generalised as it is not just Strimzi that behaves in that way. How about:

```suggestion
NOTE: If the target cluster exposes a bootstrap endpoint (as is the case with some cloud-based Kafka services or clusters managed by Strimzi), specify its bootstrap endpoint in preference to enumerating the individual broker addresses.
```